### PR TITLE
fix margin for network graph controls

### PR DESCRIPTION
**Pull request by andreasn**
Fixes issue #2831

### CODE REVIEW

File: pkg/base1/cockpit.css [hunk 1]
```diff
@@ -535,3 +535,9 @@ a {
     background-color: #888 !important;
     color: white !important;
 }
+
+/* Fix padding for network graph controls */
+
+#networking-graph-toolbar {
+    margin-right: 20px;
```
Comment (stefwalter) [inline pkg/base1/cockpit.css]: This change should go into networking.css, right?

**Comment by andreasn** [comment]
hups!
You're right. I'll put it there

**Comment by stefwalter** [comment]
This needs a rebase (even though github doesn't say so).

**Comment by andreasn** [comment]
Ugh, this rebasing business failed. I'm too bad at git.
Created #2875 instead.

**Pull request closed without merging**